tests/test_pubkey_cmd.py

```diff
@@ -33,6 +33,13 @@ def test_get_public_key_confirm_ok(backend, firmware, navigator, test_name):
                        NavInsID.USE_CASE_ADDRESS_CONFIRMATION_CONFIRM,
                        NavInsID.USE_CASE_STATUS_DISMISS]
             navigator.navigate_and_compare(ROOT_SCREENSHOT_PATH, test_name, nav_ins)
+        elif backend.firmware.device == "flex":
+            nav_ins = [NavInsID.SWIPE_CENTER_TO_LEFT,
+                       NavIns(NavInsID.TOUCH, (100,400)),
+                       NavInsID.USE_CASE_ADDRESS_CONFIRMATION_EXIT_QR,
+                       NavInsID.USE_CASE_ADDRESS_CONFIRMATION_CONFIRM,
+                       NavInsID.USE_CASE_STATUS_DISMISS]
+            navigator.navigate_and_compare(ROOT_SCREENSHOT_PATH, test_name, nav_ins)
```

Comment:
`scenario_navigator.address_review `could be used here because it's a really standard use case (no need to provide instruction, same command for all devices)

Reply:
Done